src/Products/PloneMeeting/ftw_labels/utils.py

```
@@ -12,15 +12,31 @@
 from zope.component import getAdapter
 
 
-def get_labels(obj, include_personal_labels=True, label_ids=[]):
+def get_labels(obj, include_personal_labels=True, label_ids=[], only_viewable=False, only_editable=False):
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)